### ISSUE TITLE
Azure Login behaviour

### ISSUE THREAD

**Issue by DanteNahuel** [PowerShell/PowerShell#12063]
```powershell
Use Login-Azaccount or Connect-AzAccount
```

# Expected behavior

```
Login Prompt Popup like previous Powershell

```

# Actual behavior

```
Console asking for using an url (microsoft.com/devicelogin) and entering a code to authenticate.

This doesn't let you cancel or exit unless you completely close the console and also is not helpful for endusers who don't pay attention to the console.
There should be a flag that lets you choose between Login types.

```

**Comment by MaximoTrinidad** [comment]
It's been this way for some time now. It's not a bug!

**Comment by vexx32** [comment]
It's also not a cmdlet managed by the PowerShell team; please submit this issue / feature request to the Az module repository so they can action it properly. 😊💖

**Comment by DanteNahuel** [comment]
> It's also not a cmdlet managed by the PowerShell team; please submit this issue / feature request to the Az module repository so they can action it properly. 😊💖

Oh, I see. So this is an Az Module feature, even if this only happens when using PS 7?

**Comment by vexx32** [comment]
Yeah, if they're using Windows PowerShell-only features in the module (e.g., certain methods of handling 2FA in some microsoft modules won't work as they rely on GUI libraries that aren't present in .NET Core as one example), they need to work around that in the module. I suspect what you're seeing is the result of that; if their workaround isn't sufficient they need to revisit it there. 🙂

**Comment by MaximoTrinidad** [comment]
A few years ago I have discussed it with no resolution. But, this was due to the way it handled authentication with Azure as @vexx32 mention. At that time, I didn't submit the issue. 

As this is an Azure PowerShell, create the issue in the following repository:
https://github.com/Azure/azure-powershell

Go ahead, create the issue and let me know so I can follow up and comment on it.

I'm a total believer that this should work using CLI for scripting automation purposes. 

:)
